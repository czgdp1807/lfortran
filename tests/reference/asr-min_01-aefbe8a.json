{
    "basename": "asr-min_01-aefbe8a",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/min_01.f90",
    "infile_hash": "4f2b6a5d17d5444ec72785deb89d7ba6cfb6e8d5dd05afde0d4ebdc4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-min_01-aefbe8a.stdout",
    "stdout_hash": "7bdd08fe45f9e340d2cd7f567cf22b15c6e5e8084d6aae22f60a11ae",
    "stderr": "asr-min_01-aefbe8a.stderr",
    "stderr_hash": "89a0a5e0b7e026a056095742cf701f20b354573d1802d74c364a0159",
    "returncode": 0
}