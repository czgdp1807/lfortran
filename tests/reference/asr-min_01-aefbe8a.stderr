warning: Different kinds of args in max0 is a non-standard extension
 --> tests/errors/min_01.f90:6:14
  |
6 |     print *, min(y, z)
  |              ^^^^^^^^^ help: ensure all arguments have the same kind to make it standard
